### DISCUSSION TITLE
Notifications style

### DISCUSSION THREAD

**Discussion by B8ightY** [AvaloniaUI/Avalonia#5553] [Q&A] [answered]
I have few questions about avalonia notifications:

1.  Is there any way to complete override fluent theme notifications?
2.  How to place notification manager to middle top of the screen?
3.  Can transitions be replaced and how?

Thanks for any answers

**Comment by maxkatz6** [comment, marked as answer]
All of these questions can be solved by overriding styles of these files:

Your setters with HorizontalAlignment,VerticalAlignment will allow to center notification manager, like here:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/WindowNotificationManager.xaml

And you can use Style.Animations to define custom transitions and restyle with you styles. I would recommend creating new template for NotificationCard with another controls names, so old styles will not applied.
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/NotificationCard.xaml

**Reply by maxkatz6** [reply]
You can find more information about styles and animations in general here:
https://avaloniaui.net/docs/styles/
https://avaloniaui.net/docs/animations/

**Reply by B8ightY** [reply]
What a brilliant answer! I managed to get it all work. Thank you

But I also found a bug described here #5574 so if you could help me it would be amazing
